portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,122 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Keith Williams Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
     <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+</head>
+
+<body>
+    <header>
+        <nav class = "nav-bar">
+            <ul class = "nav-list">
+                <li>
+                    <a href="#top" class="nav-link">Home</a>
+                </li>
+                <li>
+                    <a href="#about" class="nav-link">About</a>
+                </li>
+                <li>
+                    <a href="#skills" class="nav-link">Skills</a>
+                </li>
+                <li>
+                    <a href="#contact" class="nav-link">Contact</a>
+                </li>
+                <li>
+                    <a href="https://drive.google.com/file/d/1VZJ9ii5GuOdE2d3M2-93fzX6cqlepykJ/view?usp=sharing" target="_blank" class="nav-link">Resume</a>
+                </li>
+            </ul>
+        </nav>
+        <div class = "intro-name">
+            <h1>Keith Williams</h1>
+            <h6>Sophomore @ Duke University</h6>
+        </div>
+    </header>
+
+    <section class="about" id="about">
+        <div class="container">
+            <div class="img-of-me">
+                <img src = images/websiteMe.jpeg>
+            </div>
+            <div class="about-details">
+                <div class="about-heading">
+                    <h1>About</h1>
+                    <h6>Myself</h6>
+                </div>
+                <p>
+                    Hi! My name is Keith and I'm a rising sophomore at Duke University.
+                    I'm intending to major in computer science. Right now, Right now, 
```

Comment:
Remove the duplicated "Right now"?

Reply:
Whoops, didn't mean to let that slip by. Thanks